todoTask/todoTask.js

```diff
@@ -1,85 +1,59 @@
-﻿// ------------------------global varibals--------------------
-let globalID = 0;
-// -------------------------functions---------------------------
-const getId = () => (globalID += 1);
+const getId = (() => {
+    let globalID = 0;
+    return () => (globalID += 1);
+})();
```

Comment:
That's the way to hide `globalId` inside a scope
https://en.wikipedia.org/wiki/Information_hiding